src/fiatconnect/ReviewScreen.tsx

```diff
@@ -465,23 +462,34 @@ function TransactionDetails({
   feeEstimate?: FeeEstimateState
 }) {
   const exchangeRates = useSelector(localCurrencyExchangeRatesSelector)!
+  const tokenInfo = useTokenInfoBySymbol(normalizedQuote.getCryptoType())
+  const usdTokenInfo = useTokenInfoBySymbol(CiCoCurrency.cUSD)!
+  const networkFee = useLocalToTokenAmount(
+    feeEstimate?.usdFee ? new BigNumber(feeEstimate?.usdFee) : new BigNumber(0),
+    usdTokenInfo.address
+  )!
```

Comment:
nit: is it possible to avoid the duplication of `networkFee` computation in both `ReceiveAmount` and `TransactionDetails`?

Reply:
done